lbrynet/core/StreamDescriptor.py

```diff
@@ -365,7 +375,7 @@ def get_stream_hash(hex_stream_name, key, hex_suggested_file_name, blob_infos):
 
 def verify_hex(text, field_name):
     for c in text:
-        if c not in '0123456789abcdef':
+        if c not in b'0123456789abcdef':
```

Comment:
that just makes the string be `bytes` instead of `unicode`. In Python 3 strings are unicode by default, so to get the original behavior it had to be converted to `bytes` explicitely.